Source/JavaScriptCore/runtime/VM.cpp

```diff
@@ -372,7 +372,12 @@ VM::VM(VMType vmType, HeapType heapType, WTF::RunLoop* runLoop, bool* success)
     // won't use this.
     m_typedArrayController = adoptRef(new SimpleTypedArrayController());
 
-    m_bytecodeIntrinsicRegistry = makeUnique<BytecodeIntrinsicRegistry>(*this);
+    {
+#if ENABLE(REFTRACKER)
+        WTF::RefTrackerLoggingDisabledScope<StrongRefTracker> disableLoggingScope;
```

Comment:
Why do we do this here?

Reply:
I found that this cache was generating a bunch of noise. But it isn't needed